.env

```diff
@@ -0,0 +1,3 @@
+CAPTCHA_PROVIDER = "reCaptcha"
+SITE_KEY = "6LeIxAcTAAAAAJcZVRqyHh71UMIEGNQ_MXjiZKhI"
```

Comment:
Is this your secret key? If so, you should rotate your secrets. Either way, it should not be in this PR.

Reply:
Apologies, I'm a current student and not as familiar with all the ways that the end goal could have been achieved. I'm also quite new to python and had issues setting env variables. (Hence why i made this suggestion) Those keys are a "Default/testing set" provided by google.